clients/vscode-hlasmplugin/README.md

```diff
@@ -11,12 +11,16 @@
 # HLASM Language Support
 HLASM Language Support is an extension that supports the High Level Assembler language. It provides code completion, highlighting and navigation features, shows mistakes in the source, and lets you trace how the conditional assembly is evaluated with a modern debugging experience.
 
-HLASM Language Support is also part of [Code4z](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.code4z-extension-pack), an all-round package that offers a modern experience for mainframe application developers, including [COBOL Language Support](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.cobol-language-support), [Explorer for Endevor](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.explorer-for-endevor), [Zowe Explorer](https://marketplace.visualstudio.com/items?itemName=Zowe.vscode-extension-for-zowe), [Debugger for Mainframe](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.debugger-for-mainframe) and [COBOL Control Flow](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.ccf) extensions.
+HLASM Language Support is also part of [Code4z](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.code4z-extension-pack), an all-round package that offers a modern experience for mainframe application developers, including extensions for language support, data editing, testing, and source code management.
 
 ## Prerequisites
 
 - There are no client or server-side prerequisites for HLASM Language Support.
 
+## Compatibility
+
+HLASM Language Support is supported on Visual Studio Code and Github Codespaces.
```

Comment:
we do not support the browser based codespaces yet.

Reply:
We support GitHub Codespaces running on VM's. We do not support [VScode for web](https://code.visualstudio.com/docs/editor/vscode-web). That should be clear from the note. For me this is fine.